test_requirements.txt

```diff
@@ -6,3 +6,4 @@ thriftpy2
 six
 netifaces
 sphinx_autodoc_typehints
+pydata-sphinx-theme>=0.8.0
```

Comment:
This contradicts https://github.com/xnd-project/rbc/pull/436 but I guess pydata-sphinx-theme is what we want. Right?

Reply:
Yes this is what we want here.